cmd/run.go

```diff
@@ -71,6 +70,10 @@ func init() {
 	utils.ExitOnError(viper.BindPFlag("geth", flagSet.Lookup("geth")))
 	utils.ExitOnError(serveCmd.MarkFlagFilename("geth"))
 
+	flagSet.String("ipcpath", "", "Path to the geth ipc file")
+	utils.ExitOnError(viper.BindPFlag("ipcpath", flagSet.Lookup("ipcpath")))
+	utils.ExitOnError(serveCmd.MarkFlagFilename("ipcpath"))
```

Comment:
not sure what this line did... the file doesn't exist at the moment of call, so if this check if it exists it will fail